terraform/main.tf

```diff
@@ -0,0 +1,82 @@
+data "aws_caller_identity" "current" {}
+
+resource "aws_dynamodb_table" "dimsio" {
+  name           = "${var.dynamodb_table_name}"
+  read_capacity  = 1
```

Comment:
Yes, throttling is a potential concern, I think we should at least make this a terraform variable for now.  I'm hoping we can get away with finding the "right balance" and seeing if that works before jumping into autoscaling.